pkgs/by-name/rc/rcp/package.nix

```
@@ -0,0 +1,33 @@
+{ lib, stdenv
+, buildPackages
+, fetchFromGitHub
+, rustPlatform
+, pkg-config
+}:
+
+rustPlatform.buildRustPackage rec {
+  pname = "rcp";
+  version = "0.4.0";
+
+  src = fetchFromGitHub {
+    owner = "wykurz";
+    repo = pname;
+    rev = "v${version}";
+    hash = "sha256-3K0Fpy8DL2BOaFJXk/acaVhqXmI3xp4gwfbMNRCZ8N8=";
+  };
+
+  cargoHash = "sha256-FwkyHMQDnz8mgKrmNDIedHosRYC9hYxIjHDOV5ZIRh4=";
+
+  checkFlags = [
+    # this test also sets setuid permissions on a test file (3oXXX) which doesn't work in a sandbox
+    "--skip=copy::copy_tests::check_default_mode"
+  ];
+
+  meta = with lib; {
+    description = "Tools to efficiently copy, remove and link large filesets.";
+    homepage = "https://github.com/wykurz/rcp";
+    changelog = "https://github.com/wyurz/rcp/releases/tag/${version}";
+    license = with licenses; [ unlicense /* or */ mit ];
```

```suggestion
    # repo has no license, but crates.io says it's MIT
    license = with licenses; [ mit ];
    mainProgram = "rcp";
```

`Unlicense` is a license, and I know this packages a bunch of programs, but as the name of the repo is `rcp`, it could be set as the main program.

---

Thanks. I added the MIT license to the github repo.